troubleshooting/tools/tracevol.py

```diff
@@ -125,27 +151,40 @@ def format_table(arg, pvc_data, table):
         table.add_row([pvcname, pvname, "", False,
                        False, False])
         return
-    # check image details present rados omap
-    pv_present, uuid_present = validate_volume_in_rados(
-        arg, image_id, pvname, pool_name)
-    image_in_cluster = check_image_in_cluster(arg, image_id, pool_name)
+    # check image/subvolume details present rados omap
+    pv_present, uuid_present = validate_volume_in_rados(arg,
+                                                        image_id,
+                                                        pvname,
+                                                        pool_name,
+                                                        is_rbd)
+    present_in_cluster = False
+    if is_rbd:
+        present_in_cluster = check_image_in_cluster(arg, image_id, pool_name)
+    else:
+        subvolname = "csi-vol-%s" % image_id
+        present_in_cluster = check_subvol_in_cluster(arg, subvolname)
     image_name = "csi-vol-%s" % image_id
     table.add_row([pvcname, pvname, image_name, pv_present,
-                   uuid_present, image_in_cluster])
+                   uuid_present, present_in_cluster])
 
 
-def validate_volume_in_rados(arg, image_id, pvc_name, pool_name):
+def validate_volume_in_rados(arg, image_id, pvc_name, pool_name, is_rbd):
     """
     validate volume information in rados
     """
-
-    pv_present = check_pv_name_in_rados(arg, image_id, pvc_name, pool_name)
-    uuid_present = check_image_uuid_in_rados(
-        arg, image_id, pvc_name, pool_name)
+    pv_present = check_pv_name_in_rados(arg,
+                                        image_id,
```

Comment:
Each arg needs to be in a new line? same comment applies to other places also.

Reply:
Looked weird to me, I changed it to single long line instead.